implementations/rust/ockam/ockam_command/src/policy.rs

```diff
@@ -0,0 +1,82 @@
+use crate::util::{extract_address_value, node_rpc, RpcBuilder};
+use crate::{help, CommandGlobalOpts, Result};
+use clap::{Args, Subcommand};
+use ockam::{Context, TcpTransport};
+use ockam_abac::{Action, Expr, Resource};
+use ockam_api::nodes::models::policy::Policy;
+use ockam_core::api::Request;
+
+const HELP_DETAIL: &str = "";
+
+#[derive(Clone, Debug, Args)]
+#[command(hide = help::hide(), after_long_help = help::template(HELP_DETAIL))]
+pub struct PolicyCommand {
+    #[command(subcommand)]
+    subcommand: PolicySubcommand,
+}
+
+#[derive(Clone, Debug, Subcommand)]
+pub enum PolicySubcommand {
+    Set {
+        /// Node on which to start the tcp inlet.
+        #[arg(long, display_order = 900, id = "NODE")]
+        at: String,
+
+        #[arg(short, long)]
+        resource: Resource,
+
+        #[arg(short, long)]
+        action: Action,
+
+        #[arg(short, long)]
+        expression: Expr,
+    },
+    Get {
+        /// Node on which to start the tcp inlet.
+        #[arg(long, display_order = 900, id = "NODE")]
+        at: String,
+
+        #[arg(short, long)]
+        resource: Resource,
+
+        #[arg(short, long)]
+        action: Action,
+    },
+}
+
+impl PolicyCommand {
+    pub fn run(self, opts: CommandGlobalOpts) {
+        node_rpc(rpc, (opts, self))
+    }
+}
+
+async fn rpc(ctx: Context, (opts, cmd): (CommandGlobalOpts, PolicyCommand)) -> Result<()> {
+    let tcp = TcpTransport::create(&ctx).await?;
```

Comment:
Nit: if you are using a single RPC per command, you can remove this and create the rpc as: `Rpc::background(ctx, &opts, &node)?;`